components/add_groups_to_team_modal/add_groups_to_team_modal.test.jsx

```diff
@@ -83,9 +86,9 @@ describe('components/AddGroupsToTeamModal', () => {
         setTimeout(() => {
             expect(instance.handleResponse).toBeCalledTimes(2);
         }, 0);
-        expect(instance.handleHide).not.toBeCalled();
+        expect(instance.handleHide).toHaveBeenCalledTimes(1);
```

Comment:
If we make above change to `linkGroupSyncable` , this can stay as it is without changing it
```suggestion
      expect(instance.handleHide).not.toBeCalled();
```